test/integrations/destinations/mp/processor/data.ts

```diff
@@ -3502,17 +3544,29 @@ export const data = [
         status: 200,
         body: [
           {
-            error:
```

Comment:
Is restriction being older than 5 days not a concern at all in /import endpoint ?

Reply:
as long as it's < 5 years old ? 
https://github.com/rudderlabs/rudder-transformer/pull/2833/files#diff-5208294c2bfab48d117a1b6c92ffd316a5efd733831419bb7ed12484d30e9a77R77

Reply:
Test case for this is already present. 
`Test 33`